lib/EventListener/BeforeTemplateRenderedEventListener.php

```diff
@@ -64,17 +73,14 @@ public function handle(Event $event): void {
             case 'dashboard.dashboard.index':
                 if($event->isLoggedIn() && $serverstyleDash) {
                     $this->addHeaderFor('dashboard');
+                    $this->addMetadata();
                 }
                 break;
             default:
                 if($event->isLoggedIn()) {
                     if($serverstyleDash) {
                         $this->addHeaderFor('dashboard');
                     }
-                } else {
```

Comment:
That is a good question. There probably was a reason (i assume it was to prevent breakage and provide a graceful fallback for "unknown" routes), but to be frank, i dont remember why i removed it in 2022. It took me a while to finish this PR :D

Reply:
I would reintroduce it. Some apps might have paths, that you can't know here. For example this might break public links